Blockzilla/AppDelegate.swift

```diff
@@ -206,13 +206,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         splashView?.animateHidden(true, duration: 0.25)
         if let url = queuedUrl {
-            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.openedFromExtension, object: TelemetryEventObject.app)
+            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.open, object: TelemetryEventObject.app, value: "extension")
```

Comment:
If this is an old probe, we can't just change the way we track it without cost - our data stream will have been interrupted.